compiler/builtins.js

```diff
@@ -867,6 +869,25 @@ export const BuiltinFuncs = function() {
     }
   };
 
+  this.__Porffor_copyMemory = {
+    params: [ Valtype.i32, Valtype.i32, Valtype.i32 ],
+    locals: [],
+    returns: [],
+    wasm: [
+      // dst
+      [ Opcodes.local_get, 1 ],
+
+      // src
+      [ Opcodes.local_get, 0 ],
+
+      // size
+      [ Opcodes.local_get, 2 ],
+
+      [ ...Opcodes.memory_copy, 0x00, 0x00 ],
```

Comment:
Why introduce `__Porffor_copyMemory` if it goes unused?

Reply:
Yeah that's supposed to be for memory allocation stuff and I kinda just accidentally built off of that.